### DISCUSSION TITLE
Official Support for Kraken Elite AIOs in V209??

### DISCUSSION THREAD

**Discussion by kaz-mp6** [Rem0o/FanControl.Releases#2886] [Q&A] [answered]
I'm a returning user of FanControl, but I am returning with new hardware. I now have an NZXT Kraken Elite 280. Can FanControl control fan/pump speeds yet? I saw it was added to LHM, but I don't know if FanControl has implemented support yet... If so, how can I add it to my control panel?

**Comment by Rem0o** [comment, marked as answer]
FanControl runs on the latest code from LHM.   NZXT support is through the "controller" checkbox in the sensor sources.

**Reply by kaz-mp6** [reply]
Do I need NZXT CAM to be uninstalled before FanControl can take over? I currently have CAM installed, but not running. Both automatic and manual setup do not control the NZXT pump (stays at >2000 RPM.

**Reply by Kquigs** [reply]
I also would love to know, because I haven't been able to find a plugin that can do this, and the newer kraken elite doesn't seem compatible. Unless I'm missing something...

**Reply by monstermatte** [reply]
@Kquigs did you find a solution?

**Reply by Kquigs** [reply]
> @Kquigs did you find a solution?

Unfortunately not. Right now I have my motherboard keeping the pump at 100% all the time

**Comment by kaz-mp6** [comment]
Hey guys, it’s been a minute since I originally posted the question. As @Rem0o mentioned, the “Controllers (various third parties)” must be check under sensors to enable NZXT pump control. My original pump had damaged pin connectors; that seemed to be the issue (for me, at least). Try it with CAM closed and the Controllers sensor checked.

**Reply by Zorast** [reply]
Ia running an NZXT Fan Controller V2 but it can't be managed with FanControl :/

https://github.com/Rem0o/FanControl.Releases/discussions/3363

**Reply by Rem0o** [reply]
https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility
